### DISCUSSION TITLE
Recording?

### DISCUSSION THREAD

**Discussion by harishruparel** [edumeet/edumeet#856] [Ideas]
What is required to add recording to this? The media can be stored to any location, local or cloud storage like dropbox or google drive!

**Comment by abakanindex** [comment]
+1
or AWS bucket

**Comment by harishruparel** [comment]
Does this sample code have the option for recording? How do I start recording/ need any change in code?

**Comment by abakanindex** [comment]
I think we have keep only one #327 issue open for this
